tests/videos/tools/test_cuts.py

```diff
@@ -0,0 +1,15 @@
+from moviepy.editor import ColorClip, concatenate_videoclips
+from moviepy.video.tools.cuts import detect_scenes
+
+
+def test_detect_scenes__red_green():
```

Comment:
I use double underscores to separate the function being tested from the type of test, to help readability